src/test/java/uk/gov/ons/ctp/integration/contcencucumber/cucSteps/cases/TestCaseEndpoints.java

```
@@ -797,4 +799,70 @@ public void cc_Advisor_selects_the(String statusSelected) {
+    queueName = rabbit.createQueue(eventType);
+    log.info("Flushing queue: '" + queueName + "'");
+    rabbit.flushQueue(queueName);
+  }
+
+  @When("CC Advisor selects the survey launch")
+  public void cc_Advisor_selects_the_launch() {
+    try {
+      log.with(caseId)
+          .info("Now putting a ModifyCaseRequestDTO on the modifyCase endpoint for this case id..");
+      final String response = requestSurveyLaunch();
+      log.info("SURVEY LAUNCH: The response from " + telephoneEndpointUrl.toString());
+      assertNotNull("SURVEY LAUNCH failure", response);
+    } catch (Exception e) {
+      fail("SURVEY launch HAS FAILED - the contact centre does not give a response code of 200");
+    }
+  }
+
+  private String requestSurveyLaunch() {
+    log.with(telephoneEndpointUrl).info("The url for requesting the survey launch");
+
+    String launchSurveyResponseString = null;
+    try {
+      launchSurveyResponseString =
+          getRestTemplate().getForObject(telephoneEndpointUrl, String.class);
+    } catch (Exception ex) {
+      System.out.println(ex.getMessage());
+    }
+
+    return launchSurveyResponseString;
+  }
+
+  @Then("a Survey Launched event is emitted to RM, which contains the launch status type")
+  public void aSurveyLaunchedEventIsEmittedToRMWhichContainsTheLaunchStatusType() {
+    log.info(
+        "Check that a SURVEY_LAUNCHED event has now been put on the empty queue, named {}, ready to be picked up by RM",
+        queueName);
+
+    String clazzName = "SurveyLaunchedEvent.class";
+    String timeout = "2000ms";
+
+    log.info(
+        "Getting from queue: '{}' and converting to an object of type '{}', with timeout of '{}'",
+        queueName,
+        clazzName,
+        timeout);
+
+    SurveyLaunchedEvent launchedEvent = null;
+    try {
+      launchedEvent =
+          rabbit.getMessage(
+              queueName, SurveyLaunchedEvent.class, TimeoutParser.parseTimeoutString(timeout));
+    } catch (Exception exception) {
```

Surely easier to not catch the exception and let the test fail

---

Removed try catch